flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-doris/src/main/java/org/apache/flink/cdc/connectors/doris/sink/DorisDataSink.java

```diff
@@ -74,7 +76,7 @@ public EventSinkProvider getEventSinkProvider() {
     }
 
     @Override
-    public MetadataApplier getMetadataApplier() {
-        return new DorisMetadataApplier(dorisOptions, configuration);
+    public MetadataApplier getMetadataApplier(Set<SchemaChangeEventType> enabledEventTypes) {
+        return new DorisMetadataApplier(dorisOptions, configuration, enabledEventTypes);
     }
```

Comment:
You need change all connectors in one PR if you bring an incompatible change in public API, we should avoid this kind of change.

Reply:
Left `DataSink` interface unchanged, and adds `default` implementation in `MetadataApplier` interface.